src/Pages/BooksPage.js

```diff
@@ -1,24 +1,29 @@
-import './BooksPage.css';
-import bookList from '../fakeBooksData';
+import { useSelector } from 'react-redux';
 import Book from '../components/Book';
-import AddBook from '../components/AddBook';
+import AddBookForm from '../components/AddBookForm';
+// import { removeBook } from '../Redux/book/book';
```

Comment:
- [ ] [ OPTIONAL]  Kindly remove all commented-out and unwanted/ unused pieces of code from your code base. It makes it look messy and cluttered.

Reply:
Thanks for pointing that, I forgot to remove!